src/router/handlers/resourceHandler.ts

```diff
@@ -71,11 +86,17 @@ export default class ResourceHandler implements CrudHandlerInterface {
         );
     }
 
-    async typeHistory(resourceType: string, queryParams: any) {
+    async typeHistory(resourceType: string, queryParams: any, userIdentity: KeyValueMap) {

```

Comment:
Don't forget instanceHistory

Reply:
For `getSearchFilterBasedOnIdentity` it only includes `operation: 'search-type' | 'search-system' | 'history-type' | 'history-system';`

Should we add `history-instance` to that list? That was the original reason why I didn't include `history-instance` in this PR.

Reply:
Added this PR to include `history-instance`
https://github.com/awslabs/fhir-works-on-aws-interface/pull/41

Reply:
Updated this PR to include filtering `history-instance`